src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,79 @@
-export const Pagination = () => {};
+import cn from 'classnames';
+
+type Props = {
+  totalItems: number;
+  itemsPerPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  totalItems,
+  itemsPerPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const totalPages = Math.ceil(totalItems / itemsPerPage);
+  const pageNumbers = [];
+
+  for (let i = 1; i <= totalPages; i += 1) {
+    pageNumbers.push(i);
+  }
```

Comment:
This loop generates an array of page numbers every time the component renders, which may impact performance if the number of pages is large. Consider memoizing this array using `useMemo` to avoid unnecessary recalculations on each render.